docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md

```diff
@@ -0,0 +1,170 @@
+
+After installing caver-js, you will need to follow these steps.
+
+**a. Set up ABI folder**
+
+Create a new file named paymentSplitter.json in the `abi` folder.
+
+**b. Create your Scripts**
+
+Create a new file named `paymentSplitter.js` in the scripts folder.
+
+**c. Execute Scripts**
+
+Paste the code below in your newly created `paymentSplitter.js` file. The code illustrates the use of caver.js to interact with the payment splitter contract previously created.
+
+First, we import caver.js and the contract ABI, which defines the functions and parameters of the payment splitter contract. Then, we initialize caver.js and the KIP17 contract by providing the contract address.
+
+Next we set the contract address, create and add keyring to enable users sign on using their own Klaytn account. Then, we call a safeMint function to mint tokens to a specified account. Next we call the release function to enable the payee to pull funds from the contract.
+
+Finally, we checked the amount released to the payee using the `released` function.
+
+```javascript title="paymentSplitter.js"
+// Interact with payment splitter contract using caver-js
+
+// Import caver.js and the paymentSplitter contract ABI
+const Caver = require('caver-js')
+const contractABI = require("../abi/paymentSplitter.json")
+
+// Initialize caver.js and the paymentSplitter contract
+const caver = new Caver('https://api.baobab.klaytn.net:8651/')
+const contractAddr = "paste contract address"
+const contract = caver.contract.create(contractABI, contractAddr);
+
+// Create and add keyring
+const keyring = caver.wallet.keyring.createFromPrivateKey('<Paste private key from Kaikas Wallet>')
+caver.wallet.add(keyring)
+
+// function to mintTokens 
+// This ensures that  KLAY is in the contract
+const mintTokens = async () => {
+    const amountPayable =  caver.utils.convertToPeb('10', 'KLAY');
+    contract.send({from: keyring.address, gas: 1500000, value: amountPayable}, 'safeMint', keyring.address)
+    .then(receipt => {
+        console.log(receipt);
+    })
+    .catch(err => {
+        console.log(err);
+    })
+}
+
+// function to pull payee's share from the contract
+// Note anyone can call this function provided it is the payee's address
+const releaseToPayee = async () => {
+    const payeeAddress =  “<PASTE PAYEE ADDRESS>”;
+    contract.send({from: keyring.address, gas: 1500000, value: 0}, 'release', payeeAddress)
+    .then(receipt => {
+        console.log(receipt);
+    })
+    .catch(err => {
+        console.log(err);
+    })
+}
+
+
+// function to get amount released to payee
+const  getAmountReleased = async (addr) => {
+    contract.methods.released(addr).call()
+    .then(result => {
+        console.log(result);
+    })
+ }
+
+
+mintTokens();
+// releaseToPayee();
+// getAmountReleased("<PASTE PAYEE ADDRESS>");
```

Comment:
extract payee address to a constant, declare at the start and pass the variable to the method.